src/BookFormModal.js

```diff
@@ -11,26 +13,31 @@ export default class BookFormModal extends Component {
             <Modal.Title>Modal title</Modal.Title>
             </Modal.Header>
             <Modal.Body>
-            <Form onSubmit={this.handleSubmit}>
-              <Form.Group className="mb-3" controlId="BookTitle">
-                <Form.Label>Book Title</Form.Label>
-                <Form.Control type="text" placeholder="Enter Book Title" />
-                </Form.Group>
-                <Form.Group className="mb-3" controlId="BookDescription">
-                  <Form.Label>Book Description</Form.Label>
-                  <Form.Control type="text" placeholder="Enter Book Description" />
-                  </Form.Group>
-                  <Form.Group className="mb-3" controlId="formCheckbox">
-                    <Form.Check type="checkbox" label="Finished" />
-                    </Form.Group>
-                    <Button variant="primary" type="submit">
-                      Submit
-                      </Button>
-                      </Form>
+            <Container>
+        <Form onSubmit={this.props.handleSubmit}>
+          <Form.Group className="mb-3" controlId="title">
+            <Form.Label>Book Title</Form.Label>
+            <Form.Control type="text" placeholder="Book Title" />
+          </Form.Group>
+          <Form.Group className="mb-3" controlId="description">
+            <Form.Label>Description</Form.Label>
+            <Form.Control type="text" placeholder="New York" />
+          </Form.Group>
+          <Form.Group className="mb-3" controlId="email">
+            <Form.Label>email</Form.Label>
+            <Form.Control type="text" placeholder="email" />
+          </Form.Group>
+          <Form.Group className="mb-3" controlId="status">
+            <Form.Check type="checkbox" label="Read?" />
+          </Form.Group>
+          <Button variant="primary" type="submit">
+            Add a Book
+          </Button>
+        </Form>
+      </Container>
```

Comment:
- We can clean up indentation for this component.You `Modal.body` starts on line 15 so Container is the child, but then `Form` is a child of the Container etc. 👍